### ISSUE TITLE
Missing image_url causes md5 exception

### ISSUE THREAD

**Issue by lmcnulty** [responsible-ai-collaborative/aiid#637]
While testing the process of adding incident reports, I've found that if I make a report with no `image_url`, then promote it to an incident, then restart the development server, I get error involving an md5 exception:

  > Error in function module.exports in ./node_modules/md5/md5.js:152
  > Illegal argument null
  > 
  > ./node_modules/md5/md5.js:152
  >
  >  ```
  >    150 |   module.exports = function (message, options) {
  >    151 |     if (message === undefined || message === null)
  >    152 |       throw new Error('Illegal argument ' + message);
  >    153 |
  >    154 |     var digestbytes = crypt.wordsToBytes(md5(message, options));
  >    155 |     return options && options.asBytes ? digestbytes :
  > ```

Grepping for uses of md5, I found that LatestIncidentReport.js hashes the image_url:

```JavaScript
      render={({ allMongodbAiidprodReports: { nodes } }) => {
        const { image_url, cloudinary_id, title, description, epoch_date_submitted, incident_id } =
          nodes[0];

        return (
          <Card className={className}>
            <Row className="g-0">
              <Col md={4}>
                <ThumbnailImg
                  publicID={cloudinary_id ? cloudinary_id : `legacy/${md5(image_url)}`}
                  transformation={fill().height(480)}
                  alt={title}
                />
              </Col>
```

When I delete the new incident and restart the dev server, the error goes away.

I think we could solve this by instead querying for the first report / incident where an image exists – and if there is none, then either use the hash of the default image, or don't render the thumbnail.

**Comment by clari182** [comment]
Hey @lmcnulty, I've been trying to reproduce this issue but with no luck. Here are the steps I did locally:
- Create a submission with image_url empty
- Promote it to an issue
- Restarted dev server locally
- Open the home page
- I was able to see the last incident report with the fallback image rendered

Maybe this was already fixed or one of my steps is wrong. Let me know if you see anything I don't. Thanks!
